packages/editor/src/components/widgets/code-editor/CodeEditor.css

```diff
@@ -1,10 +1,10 @@
 .code-editor .code-input {
   border-radius: var(--border-radius);
-  border: var(--input-border);
-  background: var(--input-bg);
+  border: var(--basic-border) !important;
```

Comment:
!important is mostly not a good idea... 
Why is this needed here?

Reply:
That's a good question; I don't know why I set this. I believe it's because it wasn't inheriting the border, but I removed it, checked, and at first glance, everything seems to be working fine even without "!important.